src/test/java/com/wikia/webdriver/elements/mercury/pages/SearchResultsPage.java

```diff
@@ -0,0 +1,74 @@
+package com.wikia.webdriver.elements.mercury.pages;
+
+import com.wikia.webdriver.common.contentpatterns.URLsContent;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.elements.mercury.components.Loading;
+import com.wikia.webdriver.elements.mercury.components.Search;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+import lombok.Getter;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class SearchResultsPage extends WikiBasePageObject {
+  @FindBy(css = ".search-results")
+  private WebElement searchResultsContainer;
+
+  @FindBy(css = ".search--no-results")
+  private WebElement noResultsContainer;
+
+  @Getter
+  private final Search search = new Search();
+
+  private static final String searchResultClass = ".search-results__list .search-results__item";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00115)